redisless/src/server/tests/mod.rs

```diff
@@ -55,6 +55,18 @@ fn test_redis_implementation() {
     let x: u32 = con.get("intkey").unwrap();
     assert_eq!(x, 11u32);
 
+    let _: () = con.set("intkeyby", "10").unwrap();
+    let _: () = con.incr("intkeyby", "10").unwrap();
+
+    let x: u32 = con.get("intkeyby").unwrap();
+    assert_eq!(x, 20u32);
+
+    let _: () = con.set("intkeybyneg", "10").unwrap();
+    let _: () = con.incr("intkeybyneg", "-5").unwrap();
+
+    let x: u32 = con.get("intkeybyneg").unwrap();
+    assert_eq!(x, 5u32);
+
```

Comment:
@pepoviola can you add a test where we increment a key that does not exist?
E.g: `let _: () = con.incr("keydoesnotexist", "42").unwrap();`

Ty

Reply:
yes, adding now :D